src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -159,13 +174,24 @@ private function displayTxt(SymfonyStyle $io, array $filesInfo): int
                 if (false !== strpos($info['message'], 'PARSE_CUSTOM_TAGS')) {
                     $suggestTagOption = true;
                 }
+
+                if ($errorAsGithubAnnotations) {
+                    $githubErrorsReporter->error($info['message'], $info['file'], $info['line']);
+                }
             }
         }
 
         if (0 === $erroredFiles) {
             $io->success(sprintf('All %d YAML files contain valid syntax.', $countFiles));
         } else {
-            $io->warning(sprintf('%d YAML files have valid syntax and %d contain errors.%s', $countFiles - $erroredFiles, $erroredFiles, $suggestTagOption ? ' Use the --parse-tags option if you want parse custom tags.' : ''));
+            $message = sprintf('%d YAML files have valid syntax and %d contain errors.%s', $countFiles - $erroredFiles, $erroredFiles, $suggestTagOption ? ' Use the --parse-tags option if you want parse custom tags.' : '');
+
+            if ($errorAsGithubAnnotations) {
+                $githubReporter->warning($message);
+                $io->write($buffer->fetch());
```

Comment:
this should use the `OutputInterface::OUTPUT_RAW` option to avoid processing any output formatting, as BufferedOutput would have processed it already.